pkgs/development/python-modules/git-annex-adapter/default.nix

```diff
@@ -22,9 +22,12 @@ buildPythonPackage rec {
       --replace "'git-annex'" "'${git-annex}/bin/git-annex'"
   '';
 
-  # TODO: Remove for next version
   patches = [
     ./not-a-git-repo-testcase.patch
+    (fetchpatch {
```

Comment:
Please add a comment explaining the reason for the patch and a link to upstream. Would also be nice for the other patches.